usr.sbin/bsdinstall/distfetch/distfetch.c

```diff
@@ -78,15 +85,21 @@ main(void)
 	bsddialog_backtitle(&conf, OSNAME " Installer");
 
 	for (i = 0; i < ndists; i++) {
```

Comment:
I'd be tempted to have a urls_paths = malloc(ndists * PATH_MAX); and then set the urls[i] to the right offset. That would simplify the error handling.
But like you said... We're exiting soon anyway...